include/deal.II/fe/fe_system.h

```diff
@@ -1071,6 +1072,8 @@ class FESystem : public FiniteElement<dim,spacedim>
    * Mutex for protecting initialization of restriction and embedding matrix.
    */
   mutable Threads::Mutex mutex;
+
+  friend class FE_Enriched<dim,spacedim>;
```

Comment:
`fe_system.unit_support_points` and `fe_system.unit_face_support_points` and `fe_system.adjust_line_dof_index_for_line_orientation_table`.